src/main/drivers/accgyro_spi_mpu6000.c

```diff
@@ -119,8 +121,25 @@ bool mpu6000ReadRegister(uint8_t reg, uint8_t length, uint8_t *data)
     return true;
 }
 
-void mpu6000SpiGyroInit(uint8_t lpf)
+void mpu6000SpiGyroInit(gyro_t *gyro, uint8_t lpf)
 {
+    uint16_t intPeriod;
+    switch(lpf) {
+        case 0:
+            intPeriod = PERIOD_HZ(8000);
```

Comment:
OK, it seems there was some ambiguity in my comment. I meant: don't use the macros to set the initial values of the variables. So do:
```
uint16_t intPeriodUs;
...
intPeriodUs = 125;
```
Totally clear.
The purpose of conversion macros is to convert values when they are being transferred between bits of the code that use different units. When you initialise variables they should be set in their natural units.

If you want to use frequency here, then you should do:
```
uint16_t intFrequencyHz;
...
intFrequencyHz = 8000;
```

As for documenting units, I've said it before, and I will repeat it - all measurement variables should be suffixed by their units (in this case `Us` or `Hz`) - that way you can easily pick out bugs when different units are used (degrees and decidegrees, IIRC, had some bugs because of incorrect units)